src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonSynchronisationService.kt

```diff
@@ -127,29 +130,30 @@ class ContactPersonSynchronisationService(
           telemetry + ("dpsPrisonerContactId" to it.dpsId),
         )
       } ?: run {
-        nomisApiService.getPerson(nomisPersonId = event.personId).also { nomisPerson ->
-          val nomisContact = nomisPerson.contacts.find { it.id == event.contactId } ?: throw IllegalStateException("Contact ${event.contactId} for person ${event.personId} not found in NOMIS")
-          val dpsPrisonerContact = dpsApiService.createPrisonerContact(nomisContact.toDpsCreatePrisonerContactRequest(nomisPersonId = event.personId)).also {
-            telemetry["dpsPrisonerContactId"] = it.id
-          }
-          val mapping = PersonContactMappingDto(
-            nomisId = event.contactId,
-            dpsId = dpsPrisonerContact.id.toString(),
-            mappingType = PersonContactMappingDto.MappingType.NOMIS_CREATED,
-          )
+        track("contactperson-contact-synchronisation-created", telemetry) {
```

Comment:
same change everywhere - everything wrapped in a `track` call that outputs the correct telemetry depending on success/exception